mithril.js

```diff
@@ -1959,9 +1988,17 @@
 		}
 	}
 
+	function isNativeError(e) {
```

Comment:
@pygy 

I don't think it would be safe to check that without exposing easy potential for XSS and CORS vulerabilities. If an error occurs in a child iframe, and it was constructed with `new`, it is `instanceof` the error's constructor within that frame. If an attacker defines a malicious getter on `window` and then throws an error that is `instanceof` the result of that getter, in hopes it gets caught, we've got a huge potential vulnerability. As a concrete example:

```js
// In the <iframe>
var oldTypeError = TypeError
Object.defineProperty(window, "TypeError", {
  get: function () {
    initVirusDownload();
    // Let's make it still seem like it's the real thing.
    return oldTypeError;
  },
  set: function (value) {
    // Let's make it still seem like it's the real thing.
    oldTypeError = value;
  },
})
```

Also, I don't know of any reliable way to get the currently executing iframe, and I don't want to. It would also open up a whole class of XSS and CORS vulnerabilities by pure necessity.

I'd prefer *not* to open that can of worms.